source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogueBox.java

```diff
@@ -132,79 +176,71 @@ private Label createLabel() {
         label.setFontScale(1.5f);
 
         updateLabelPosition();
-
-        return label;
     }
 
     /**
      * Moves the labels position ot fit the dialogue box
      */
     private void updateLabelPosition() {
-        float middle = stage.getWidth();
-        float labelWidth = label.getPrefWidth();
-        float labelHeight = label.getPrefHeight();
-        float labelX = (middle - labelWidth) / 2;
-        float labelY = -210 + (backgroundImage.getHeight() - labelHeight) / 2;
-        label.setPosition(labelX, labelY, Align.center);
+        float desiredHeight = screenHeight * 0.10f;  // Background image height
+        float newWidth = screenWidth * 0.9f;  // Background image width
+        label.setSize(newWidth, desiredHeight);
+        label.setPosition((screenWidth - label.getWidth()) / 2, screenHeight * 0.08f);
+        label.setAlignment(Align.center);
     }
 
     /**
      * Creates the forward button for navigating to the next hint.
-     *
-     * @return The TextButton instance.
      */
-    private TextButton createForwardButton() {
-        TextButton.TextButtonStyle buttonStyle = createButtonStyle();
-        TextButton forwardButton = new TextButton("Continue", buttonStyle);
-        forwardButton.getLabel().setAlignment(Align.center);
+    private void createForwardButton() {
```

Comment:
I believe I have fixed this, would you mind taking another look please?

Reply:
looks good, very occasionally (i cant replicate it just happens sometimes) when you hit continue it restarts the dialogue. I think its because of walking into them again while the dialogue is still up, but I'm not 100% sure. not sure if this was intended but your change also lets you get rid of the dialogue box for items which is nice.

Reply:
I know why its happened but I am unsure of who is causing it. It am imagining the bananas hitting our entity is triggering some event. This is something I did notice myself but I need to figure out who is triggering some event on the FNPC. 

If you wanted to know, in PauseEvent it relies on the last event being called is the PauseStart[AnimalName]. If this wasn't the last event called on the FNPC it will open the dialogue again from the start.